### PR TITLE
Drop support for python3.7 and bring support for django3.12

### PR DESCRIPTION
**Pull request by ticosax**
Also test against django releases 4.2 & 5.0
And change dependency to [django-fsm-2](https://github.com/pfouque/django-fsm-2) a maintained fork of django-fsm.

### CODE REVIEW

**Comment by pfouque** [comment]
> @pfouque are you okay with this change ?

Definitely!

NB: The repository will probably be transferred to [django-commons](https://github.com/django-commons) at some point, but it will be transferred with access to the PyPI project!

File: tox.ini [hunk 1]
```diff
@@ -22,4 +23,6 @@ setenv=
 deps =
     dj-3.2: Django>=3.2,<3.3
     dj-4.1: Django>=4.1,<4.2
```
Comment (pfouque) [inline tox.ini:25]: These versions are not officially supported anymore, we could drop them...

**Review by pfouque — APPROVED** [review]
LGTM! Thanks!

File: tox.ini (hunk 1 above)
Comment (ticosax) [reply tox.ini:25]: Thanks will do !

**Review by lorenzomorandini — APPROVED** [review]

**Comment by ticosax** [comment]
image
merging is blocked by this requirement. But I don't know how to disable it.
Any advice ?

**Comment by lorenzomorandini** [comment]
@ticosax do you have access to the settings of the repo? I don't but this could be it https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule#deleting-a-branch-protection-rule

**Comment by pfouque** [comment]
You could try pushing an empty commit to force a new run of those checks ... It may work :)

**Comment by ticosax** [comment]
> @ticosax do you have access to the settings of the repo? I don't but this could be it https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule#deleting-a-branch-protection-rule

unfortunately I don't have access to the settings. @MRigal maybe yourself ?

**Comment by ticosax** [comment]
> You could try pushing an empty commit to force a new run of those checks ... It may work :)

I rebased the branch and it's still the same. Apparently like @lorenzomorandini suggested, it is a branch protection rule.
We will have to call for help with one of the roadies I guess?

**Comment by jezdez** [comment]
This is now finally unblocked, also see my comment in https://github.com/jazzband/help/issues/368

**Pull request merged**